README.md

```diff
@@ -13,80 +13,36 @@ Angular package.
+`angular_gallery_section` package.
 
-The pub transformer has been removed in favor of code generation through package
-[build]. Please see the Dart 2 [migration guide].
+## [angular_gallery_section]
 
-[build]: https://pub.dartlang.org/packages/build
-[migration guide]: https://webdev.dartlang.org/dart-2
+Code generators that create a gallery page containing documentation and examples
+from a `@GallerySectionConfig` annotation.
 
-## Useful links
+## [examples]
 
-* **[AngularDart Components Gallery](https://dart-lang.github.io/angular_components/)**
-* AngularDart Components resources:
-  * [About AngularDart Components](https://webdev.dartlang.org/components)
-  * [Google I/O 2017 codelab](https://codelabs.developers.google.com/codelabs/your-first-angulardart-web-app)
-  * [AngularDart Components launch codelab](https://webdev.dartlang.org/codelabs/angular_components)
-  * [API reference](https://webdev.dartlang.org/components/api)
-* AngularDart resources:
-  * [AngularDart documentation](https://webdev.dartlang.org/angular/guide)
-  * [angular pub package](https://pub.dartlang.org/packages/angular)
-* [Material Design site](https://material.io)
+Example packages for each of the components. Additionally, the
+[examples/angular_components_example] subdirectory contains the base package
+for building the AngularDart gallery.
 
-## Required Icon Font
-
-For icons to appear, add the following icon font download to your page.
-[Example](https://github.com/dart-lang/angular_components_example/blob/ae323a593cbb5508fa6fbe33afb7f5d40ea96a7e/angular_gallery/lib/builder/template/index.html.mustache#L8)
-
-```html
-<link
-  rel="stylesheet"
-  type="text/css"
-  href="https://fonts.googleapis.com/icon?family=Material+Icons">
-```
-
-## Custom component styles
-
-The styles of these components can be customized in your project with a
-component's provided [Sass mixins](http://sass-lang.com/guide#topic-6).
-
-1. Add a dev dependency on the
-[sass_builder](https://pub.dartlang.org/packages/sass_builder) package to your
-`pubspec.yaml`:
-
-  ```yaml
-  dev_dependencies:
-    sass_builder: ^2.0.0 # update for the latest version
-  ```
-
-2. Now you can import styles and mixins from this package via dart style package
-imports in your Sass files.
-
-  In your `.scss` files:
-
-  ```scss
-  @import 'package:angular_components/css/material/material';
-
-  .blue {
-    color: $mat-blue;
-  }
-  ```
+[angular_components]: (https://github.com/dart-lang/angular_components/tree/master/angular_components)
+[angular_gallery]: (https://github.com/dart-lang/angular_components/tree/master/angular_gallery)
+[angular_gallery_section]: (https://github.com/dart-lang/angular_components/tree/master/angular_gallery_section)
+[examples]: (https://github.com/dart-lang/angular_components/tree/master/examples)
+[example/angular_components_example]: (https://github.com/dart-lang/angular_components/tree/master/examples/angular_components_example)
```

Comment:
This link isn't showing up above because of a missing "s" in "examples":

- [example/angular_components_example] -> [example<ins>s</ins>/angular_components_example]

Reply:
Thanks!